docs/Optimization.md

```diff
@@ -95,3 +95,34 @@ Example using the RedisCache adapter with the Memory adapter and a TTL of 4800 s
   adapter = Flipper::Adapters::RedisCache.new(memory_adapter, redis, 4800)
   flipper = Flipper.new(adapter)
 ```
+
+### CacheStore
+
+Rails applications can cache Flipper calls in any [ActiveSupport::Cache::Store](http://api.rubyonrails.org/classes/ActiveSupport/Cache/Store.html) implementation.
```

Comment:
of course any application using ActiveSupport::Cache could use this adapter, but I would think most people would be working on Rails applications.  Should this say something along the lines of *Applications using ActiveSupport...* or do you think the above works well?